ignite/pkg/cache/cache.go

```diff
@@ -0,0 +1,148 @@
+package cache
+
+import (
+	"bytes"
+	"encoding/gob"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+const dbName = "ignite_cache.db"
```

Comment:
```suggestion
```

I would suggest we remove this one entirely, this way use can customize the whole path and `pkg/*` can be unaware of Ignite specific logic.

Reply:
Good point. Fixed.